### DISCUSSION TITLE
Consiglio Condizionatori Smart che non richiedano il Cloud o app dedicate

### DISCUSSION THREAD

**Discussion by lcorsini** [indomus/forum#2601] [Hardware & Firmware]
Ciao, ho un paio di condizionatori samsung che iniziano ad avere una certa età ma hanno il pregio di avere una interfaccia SOAP particolarmente facile da utilizzare in locale e da interfacciare (con un toolettino chiamato hvac_ip_mqtt_bridge) con home assistant via MQTT

il problema è che, come dicevo iniziano ad avere quasi 10 anni quindi sto iniziando a capire con cosa sostituirli e chiedo al forum se qualcuno ha esperienza di device integrabili con homeassistant, i mei unici requisiti sono:

- Accesso completamente locale, niente cloud

- Possibilimente configurabili senza bisogno di app dedicate o creazione di account (questo è un problema minore cmq, basta che poi non richiedano l'account per funzionare)

Mi va bene anche qualcosa di DIY per quanto riguarda la parte di connettività, avevo visto qualcosa per daikin, mi pare, in passato, fatto con degli esp8266

**Comment by KC-inDomus** [comment]
secondo me dimentichi il più importante dei requisiti: che siano il top come climatizzatori, e al diavolo se non hanno supporto per l'integrazione. alle brutte usi smartir che funziona benissimo.

**Reply by lcorsini** [reply]
quello diciamo che lo do per scontato, o meglio, chiedo info in un forum di domotica per quanto riguarda l'integrazione :D le cose fatte alle brutte tendenzialmente non mi piacciono, ho provato in passato qualche dispositivo ad infrarossi  broadlink e non mi sono piaciuti piu che altro perchè non ho un feedback dal condizionatore, in questo periodo mi è capitato spesso di accenderli da remoto, sia per i gatti, sia per il mio ritorno a casa e voglio poter sapere dal condizionatore se è acceso e qual'è il suo stato.

**Reply by indomus** [reply]
con smartir hai stato e tutto, e funziona egregiamente. il consiglio cmq resta valido (non solo per te ma per chi eventualmente leggerà in futuro): prima le prestazioni, poi semmai l'integrabilità.

**Comment by giannicarabelli** [comment]
https://github.com/geoffdavis/esphome-mitsubishiheatpump

io uso il cloud di mitsubishi. Ma l'interfaccia fai da te era il mio piano B.
